pr-deploy/src/github.ts

```diff
@@ -32,8 +45,8 @@ export class PullRequest {
   private github: Octokit;
   private headSha: string;
 
-  constructor(github: GitHubAPI, headSha: string) {
-    this.github = github as unknown as Octokit;
+  constructor(github: Octokit, headSha: string) {
+    this.github = github as Octokit;
```

Comment:
Since `github` is already declared to be of type `Octokit`, is this `as Octokit` clause still necessary?